specs/2019.json

```diff
@@ -18622,7 +18622,7 @@
       "asiaImpairmentScale":"B",
       "rightSensoryZpp":"NA",
       "leftSensoryZpp":"NA",
-      "rightMotorZpp":"T12",
+      "rightMotorZpp":"T12*",

```

Comment:
After updating the logic, this case started breaking. Values should have a `*`. I reviewed this case with Kris Walden.